src/Forum/Content/Index.php

```diff
@@ -77,6 +78,8 @@ public function __invoke(Document $document, Request $request)
             'page' => ['offset' => ($page - 1) * 20, 'limit' => 20]
         ];
 
+        $params['filter'] = array_merge($filters, $params['filter']);
```

Comment:
Since $params['filter'] is empty here, let's just set `'filter' => $filters` in the `$params = ...` code above?

Reply:
Done :)